contrib/win32/openssh/OpenSSHTestHelper.psm1

```diff
@@ -126,7 +130,7 @@ WARNING: Following changes will be made to OpenSSH configuration
         Write-Warning $warning
         $continue = Read-Host -Prompt "Do you want to continue with the above changes? [Yes] Y; [No] N (default is `"Y`")"
         if( ($continue -eq "") -or ($continue -ieq "Y") -or ($continue -ieq "Yes") )
```

Comment:
I can change it to elseif ( ($continue -ne "") -and ($continue -ine "Y") -and ($continue -ine "Yes") )
but will do it in next PR to avoiding rerun all tests this time.